File: Onoicrm.Domain/Entities/ToothState.cs
namespace Onoicrm.Domain.Entities;

public class ToothState : Entity
{
    public string Caption { get; set; } = "";
    public string Description { get; set; } = "";
    public string CssClass { get; set; } = "";
    public string Color { get; set; } = "";
    public long? ClinicId { get; set; }
    public Clinic? Clinic { get; set; }
    protected override int GetClassId() => ClassNames.ToothState;
}

File: Onoicrm.Front/src/modules/tooth/entities/ToothState.ts
import {getGUID} from "~/shared/lib/getGUID";

export class ToothState {
  public id:string = getGUID();
  public cssClass!: string;
  public caption!: string;
  public description!: string;
  public color!:string;
  constructor(init?: Partial<ToothState>) {
    Object.assign(this, init)
  }
}


